chemprop/uncertainty/calibrator.py

```diff
@@ -68,20 +66,73 @@ def apply_calibration(self, preds, uncs) -> Tensor:
 
 @UncertaintyCalibratorRegistry.register("platt")
 class PlattCalibrator(UncertaintyCalibrator):
-    def calibrate(self, preds, uncs, targets, mask) -> Tensor:
-        ...
-        return
-
-    def apply_calibration(self, preds, uncs) -> Tensor:
-        ...
-        return
+    """A calibration method for classification datasets based on the Platt scaling algorithm, as
+    discussed in [guo2017]_ and [platt1999]_. In Platt's paper, he suggests using the number of
+    positive and negative examples in the dataset used to train the model to adjust the value of
+    target probabilities used to fit the parameters.
+
+    References
+    ----------
+    .. [guo2017] Guo, C.; Pleiss, G.; Sun, Y.; Weinberger, K. Q. "On calibration of modern neural
+        networks". ICML, 2017. https://arxiv.org/abs/1706.04599
```

Comment:
```suggestion
    .. [guo2017] Guo, C.; Pleiss, G.; Sun, Y.; Weinberger, K. Q. "On calibration of modern neural
    networks". ICML, 2017. https://arxiv.org/abs/1706.04599
```
When using a line break to continue a paragraph (where a "paragraph" can be just a single line, as it is here), don't indent subsequent lines. Yes, it looks weird in raw RST formatting, but the output formatting will be correct.